gnd/src/main/java/com/google/android/gnd/MainViewModel.java

```diff
@@ -50,11 +51,13 @@ public MainViewModel(
     this.navigator = navigator;
 
     // TODO: Move to background service.
-    disposeOnClear(
-        projectRepository
-            .getActiveProjectOnceAndStream()
-            .switchMapCompletable(this::syncFeatures)
-            .subscribe());
+    // Subscribe to the project repository
+    Disposable projectSubscription = projectRepository.getActiveProjectOnceAndStream()
+      .switchMapCompletable(this::syncFeatures)
+      .subscribe();
+
+    // Ensure that the subscription is disposed of when the model is cleared
+    disposeOnClear(projectSubscription);
```

Comment:
This will be a frequent enough usage of RxJava that we may not want to make this explicit on each usage; it's part of understanding that Reactive steam subscriptions must be cleaned up.